UN-OCHA/rwint9-site
# [RW-514] Remove migration code - part 1

Refs: RW-514

This removes the migration code from the reliefweb modules (except the reliefweb_migrate one) and the migration config files. It also disables the migration modules.

Once deployed, the migration modules can be fully removed (see https://github.com/UN-OCHA/rwint9-site/pull/507).

### Tests

1. Check out the branch
2. Run `drush cim`
3. Test different workflows (creating/editing/deleting content etc.)
